### PR TITLE
Uniswap Forks: Fixing issue #657

### PR DESCRIPTION
**Pull request by steegecs**
Simply removed the assignment of zeros to the rewardTokenEmissionsAmount and rewardTokenEmissionsUSD from pool instantiation.

### CODE REVIEW

File: subgraphs/uniswap-forks/src/price/price.ts [hunk 1]
```diff
@@ -218,7 +218,7 @@ export function getTrackedVolumeUSD(
   ) {
     return [
       tokenAmount0.times(price0USD),
-      BIGDECIMAL_ZERO,
+      tokenAmount0.times(price0USD),
```
Comment (this-username-is-taken) [inline subgraphs/uniswap-forks/src/price/price.ts:221]: Is change for addressing #657 as well or for a separate issue?

Comment (steegecs) [reply subgraphs/uniswap-forks/src/price/price.ts]: I did not mean to change this in this PR. I went ahead and removed it.

File: README.md [hunk 2]
```diff
@@ -68,25 +68,25 @@ This repo contains subgraphs defined using a set of standardized schemas. These
 | Protocol | Status | Versions † | Deployments |
 | ------- | :------: | --- | --- |
 | **DEX AMM** | | |
-| [Apeswap](https://apeswap.finance/) | ✅ | 1.3.0 / 1.1.3 / 1.0.0 | [![Apeswap BSC](./docs/images/chains/bsc.png)](https://thegraph.com/hosted-service/subgraph/messari/apeswap-bsc) [![Apeswap Polygon](./docs/images/chains/matic.png)](https://thegraph.com/hosted-service/subgraph/messari/apeswap-polygon) |
+| [Apeswap](https://apeswap.finance/) | 🛠 | 1.3.0 / 1.1.4 / 1.0.0 | [![Apeswap BSC](./docs/images/chains/bsc.png)](https://thegraph.com/hosted-service/subgraph/messari/apeswap-bsc) [![Apeswap Polygon](./docs/images/chains/matic.png)](https://thegraph.com/hosted-service/subgraph/messari/apeswap-polygon) |
```
Comment (this-username-is-taken) [inline README.md:71]: I think we can leave this as prod-ready, as the fix is minor

Comment (steegecs) [reply README.md]: Set to production ready.

**Pull request merged**